Client/Application/AppDelegate.swift

```diff
@@ -484,6 +484,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 self.launchFromURL(params)
             }
         }
+
+        guard let extensionDefaults = UserDefaults(suiteName: "group.org.mozilla.ios.Fennec") else { return }
```

Comment:
By this hack I mean communicating via settings. We need a better way for this whole patch. Or just not count bookmarks that have been added via the share extension for now. cc @Sdaswani